test/e2e/discovery/service_discovery.go

```diff
@@ -61,14 +61,14 @@ func RunServiceDiscoveryTest(f *lhframework.Framework) {
 		f.AwaitServiceImportIP(framework.ClusterA, nginxServiceClusterB)
 	}
 
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, superclusterDomain, true)
+	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, clustersetDomain, true)
 
 	f.DeleteServiceExport(framework.ClusterB, nginxServiceClusterB.Name, nginxServiceClusterB.Namespace)
 	f.AwaitServiceImportDelete(framework.ClusterA, nginxServiceClusterB.Name, nginxServiceClusterB.Namespace)
 
 	f.DeleteService(framework.ClusterB, nginxServiceClusterB.Name)
 
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, superclusterDomain, false)
+	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, clustersetDomain, false)
```

Comment:
Can we try both "clusterset" and "supercluster" in verifyServiceIpWithDig and revert it once the operator/helm patch goes in?

Reply:
Yes, done!

Reply:
@skitt oops, I think the helm patch only configured the new clusterset, and not the "compatibility" one, so it will fail.

Reply:
@mangelajo what will fail? If you’re referring to the e2e tests, they should be OK; the tests pass if either one of the domains is available.